posthog/api/test/dashboards/__snapshots__/test_dashboard.ambr

```diff
@@ -2575,6 +2615,7 @@
          "posthog_team"."ingested_event",
          "posthog_team"."autocapture_opt_out",
          "posthog_team"."autocapture_exceptions_opt_in",
+         "posthog_team"."autocapture_exceptions_errors_to_ignore",
```

Comment:
cc: @pauldambra was this left out somehow by the job in the last PR?

Reply:
Oh, I've seen this recently with some dashboard changes where snapshots in master seemed to be "out of sync" (for want of a better word) with PRs

https://github.com/PostHog/posthog/pull/16143

I didn't make time to track down the issue